docs/user_guide/vis/index.rst

```diff
@@ -1,88 +1,195 @@
+.. _IFV:
 
-Visualization
-=============
+IDAES Flowsheet Visualizer
+===========================
 
-.. warning::
-    The visualization tool is still in active development and we
-    hope to improve on it in future releases. Please use its
-    functionality at your own discretion.
+.. contents::
+    :depth: 2
 
-Overview
---------
-
-The Flowsheet Visualizer is a service that starts a flask server to
-display an interactive webpage with the current flowsheet's unit models and
-arcs. Users may manipulate the display by clicking and dragging the unit 
-models, streams, and stream labels.
+Introduction
+------------
+The IDAES Flowsheet Visualizer, or IFV for short, is a web-based user interface (UI) that lets you:
 
-Installation instructions
--------------------------
+* View any IDAES flowsheet as a process engineering diagram
+* Export flowsheet diagrams as images
+* View the "stream table" for the flowsheet
+* Rearrange the flowsheet diagram to your taste and save the arrangement for next time
+* Dynamically refresh the displayed values to reflect changes in the underlying IDAES model
 
-1. :ref:`Ensure that the latest IDAES is installed. <getting_started/index:Installation>` 
+The IFV can be invoked from a Jupyter Notebook or a Python script. It does not require that you run any
+other "server" application. Currently the IFV is only for viewing the flowsheet on your own computer (but,
+since it is a web application, a shared service for viewing flowsheets stored remotely is definitely possible).
 
-.. _usage:
+Starting and stopping the IFV is fast and does not consume many resources, a fact which we hope encourages
+its frequent use.
 
-Usage
+Guide
 -----
+This guide describes how to invoke and use the IFV.
+
+Invocation
+^^^^^^^^^^
+Once you have created your flowsheet, simply call the `visualize` method on that object,
+passing some parameters to give it a name and optional file for saving changes::
+
+    # First, create your IDAES model, "m", which has an attribute ".fs" for the flowsheet
+    # Then, invoke the `visualize` method
+    m.fs.visualize("My Flowsheet", save_as="my_flowsheet.json")
+
+The invocation of the `visualize` method will pop up a browser tab or window with the UI, displaying the
+flowsheet and, if the information is available, the stream table. In the notebook or script, you can continue
+to run more code and the UI will continue to work in the background. You can close the UI at any time. If you
+exit the script or notebook while the UI is running, it will continue to function but will not be able to save
```

Comment:
Instead of "it will continue to function" maybe say "you can continue to manipulate the diagram and stream table"